libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CMakeLists.txt

```diff
@@ -29,4 +29,8 @@ target_include_directories(${TARGET}
 
 target_link_libraries(${TARGET}
     ${PROJECT_NAME}_transisthor
-)
\ No newline at end of file
+    CONAN_PKG::sfml
+)
+
+include(${PROJECT_SOURCE_DIR}/build/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```

Comment:
This is impossible to include the SFML here. This library is linked with the server part.